src/runner/kafka.ts

```diff
@@ -167,10 +155,13 @@ class KafkaRunner extends BaseRunner
 
   consumeCallback = async (err, messages) => {
     this.logger.debug('Consumer callback');
-    await this.checks(
-      () => {},
-      () => this.healthCheck
-    );
+    await this.healthCheck();
+
+    if (this.steveo.exiting) {
```

Comment:
We consume the message but don't commit, so it should be fine right?

There is no real hook into the kafka client to gracefully stop processing messages from what I can see

Reply:
Yup, as long as we haven't committed we're fine.